articles/other/other-how-use-webhooks.md

```diff
@@ -0,0 +1,115 @@
+
+## Overview
+
+The [UKCloud Service Status page]([status](https://status.ukcloud.com)) enables you to subscribe to updates using webhook notifications. To use these notifications, you need to set up an endpoint to receive and process these notifications. For more information about webhooks, see the Atlassian Statuspage documentation: [Webhook Notifications](https://help.statuspage.io/help/webhook-notifications).
+
+This article provides sample PHP code that processes webhook HTTP POST updates from the UKCloud Service Status page in a JSON object.
+
+> [!IMPORTANT]
+> Input directly from the internet should never be trusted! Before implementing this code please ensure you have safely sanitized all user input and taken the relevant security precautions to prevent yourself from being hacked. UKCloud takes no responsibility for any damage caused to any system by use of this code.
+
+After you upload the file containing your code to your public web server, you can use the endpoint to subscribe to webhook notifications on the UKCloud Service Status page. For more information, see [*How to view UKCloud service status information*](other-how-view-status-info.md).
+
+![Webhook notifications](images/other-status-webhook.png)
+
+## Sample code
+
+```php
+<?php
+
+//Input directly from the internet should never be trusted!
+//Before implementing this please ensure you have safely sanitized all user input
+//and taken the relevant security precautions to prevent yourself from being hacked.
+//UKCloud takes no responsibility for any damage caused to any system by use of this code.
+
+$input = file_get_contents('php://input');
+
+$statusObject = json_decode($input);
+
+if(json_last_error() !== JSON_ERROR_NONE){
+  die('Invalid JSON');
+}
+
+if(!isset($statusObject->page->status_description)){
+  die('Invalid Input');
+}
+
+//Status page general information
+echo "<br><br>Status Description: " . $statusObject->page->status_description;
+echo "<br>Status Indicator: " . $statusObject->page->status_indicator;
+
+//The below if statement catches component status changes
+if(isset($statusObject->component_update)){
+  echo "<br><br>Component Name: " . $statusObject->component->name;
+  echo "<br>Old Status: " . $statusObject->component_update->old_status;
+  echo "<br>New Status: " . $statusObject->component_update->new_status;
+}
+
+//The below if statement catches incident status changes
+if(isset($statusObject->incident)){
+  echo "<br><br>Incident Name: " . $statusObject->incident->name;
+  echo "<br>Incident Impact: " . $statusObject->incident->impact;
+  echo "<br>Incident Status: " . $statusObject->incident->status;
+  echo "<br>Created At: " . $statusObject->incident->created_at;
+  echo "<br>Updated At: " . $statusObject->incident->updated_at;
+  echo "<br>Monitoring At: " . $statusObject->incident->monitoring_at;
+
+  echo "<br><br>Incident Updates: ";
+
+  //The below foreach loop goes through each component status update
```

Comment:
missing space? or this standard command?

Reply:
standard command